### PR TITLE
Change italic method to em in Change Text jQuery

### LINKED ISSUE #7512: Outdated Italicize method
Challenge [Change Text Inside an Element Using jQuery](http://www.freecodecamp.com/challenges/change-text-inside-an-element-using-jquery) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.2; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.
In the descriptionof the challenge, it uses `<i>` to replace the italicized text with jquery instead of the `<em>` tag. In a previous challenge it mentions how `<i>`is outdated and usually used for icons in bootstrap now

ie.
`$("h3").html("<i>jQuery Playground</i>");`
should be:
`$("h3").html("<em>jQuery Playground</em>");`

My code:
```html
<script>
  $(document).ready(function() {
    $("#target1").css("color", "red");

  });
</script>

<!-- Only change code above this line. -->

<div class="container-fluid">
  <h3 class="text-primary text-center">jQuery Playground</h3>
  <div class="row">
    <div class="col-xs-6">
      <h4>#left-well</h4>
      <div class="well" id="left-well">
        <button class="btn btn-default target" id="target1">#target1</button>
        <button class="btn btn-default target" id="target2">#target2</button>
        <button class="btn btn-default target" id="target3">#target3</button>
      </div>
    </div>
    <div class="col-xs-6">
      <h4>#right-well</h4>
      <div class="well" id="right-well">
        <button class="btn btn-default target" id="target4">#target4</button>
        <button class="btn btn-default target" id="target5">#target5</button>
        <button class="btn btn-default target" id="target6">#target6</button>
      </div>
    </div>
  </div>
</div>

```

### PR DESCRIPTION
**Pull request by erictleung**
Tested locally. Passes `npm run test-challenges`. Closes #7512.
- Change italic example from using i to em tag
- Add spaces in editor for ease of code addition
- Update tests to reflect em tag change

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -556,7 +556,7 @@
         "fccss",
         "  $(document).ready(function() {",
         "    $(\"#target1\").css(\"color\", \"red\");",
-        "",
+        "    ",
```
Comment (hallaathrad) [inline seed/challenges/01-front-end-development-certification/jquery.json]: is this space necessary?

**Comment by hallaathrad** [comment]
lgtm :+1:

File: seed/challenges/01-front-end-development-certification/jquery.json (hunk 1 above)
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/jquery.json]: From a user perspective, it lines up the line of code you're about to write with the previous line. So it is not really *necessary* per say, but I think it helps on the user end. To me, it's kinda like if your text editor or word processor keeps the indent of your text after you make a new line.

**Comment by sludge256** [comment]
This also resolves https://github.com/FreeCodeCamp/FreeCodeCamp/issues/5425

**Pull request merged**